app/src/components/submissionPage/GetExtraLength.test.tsx

```diff
@@ -0,0 +1,29 @@
+import MentionElements from "../../types/MentionElements";
+import GetExtraLength from "./GetExtraLength";
+
+describe("<SubmissionTextBox />", () => {
```

Comment:
Again, this isn't testing `<SubmissionTextBox />`!

Reply:
yup :/